Fix conversion warning

```
nuklear_gamepad.h: In function ‘nk_gamepad_button’:
/home/rob/Documents/nuklear_console/test/../vendor/nuklear_gamepad/nuklear_gamepad.h:373:41: warning: conversion to ‘unsigned int’ from ‘int’ may change the sign of the result [-Wsign-conversion]
  373 |         gamepads->gamepads[num].buttons &= ~NK_GAMEPAD_BUTTON_FLAG(button);
```

--- a/nuklear_gamepad.h
+++ b/nuklear_gamepad.h
@@ -371,7 +371,7 @@ NK_API void nk_gamepad_button(struct nk_gamepads* gamepads, int num, enum nk_gam
         gamepads->gamepads[num].buttons |= NK_GAMEPAD_BUTTON_FLAG(button);
     }
     else {
-        gamepads->gamepads[num].buttons &= ~NK_GAMEPAD_BUTTON_FLAG(button);
+        gamepads->gamepads[num].buttons &= (unsigned int)~NK_GAMEPAD_BUTTON_FLAG(button);
     }
 }
 

